src/appMain/smartDeviceLink.ini

```diff
@@ -211,6 +211,10 @@ ApplicationListUpdateTimeout = 2000
 ThreadPoolSize = 1
 ; The max size of hash which is used by OnHashUpdated
 HashStringSize = 32
+; Menu title default text
+MenuTitle = MENU
+; Path to default menu icon
+MenuIcon = 
```

Comment:
@dtrunov This is a requirement of the CRQ: 
MenuIcon default value should be empty (APPLINK-22707)